pkgs/misc/tmux-plugins/default.nix

```diff
@@ -70,8 +70,8 @@ in rec {
     src = fetchFromGitHub {
       owner = "tmux-plugins";
       repo = "tmux-continuum";
-      rev = "1531b3770a7cf7373d15fedd239c5331b99342d1";
-      sha256 = "1w3f7gzvv1k25yfr6d1snr2z88p8f87cahrbaslmyphdxpy0fa4m";
+      rev = "26eb5ffce0b559d682b9f98c8d4b6c370ecb639b";
+      sha256 = "1z4zx15y0lbkrjapzszg8mhb86i5p3ark71vsl3a3pwb7cx8c3fa";
     };
     dependencies = [ resurrect ];
```

Comment:
:medal_sports: Bonus point to complete the `meta` section of this package ([example](https://github.com/nixos/nixpkgs/blob/39e62d5d6e26668a818fdbdae46c4f571447f4e5/pkgs/tools/misc/tmux/default.nix#L57-L79)).

Most importantly the following attributes
```
homepage
description
license
platforms
maintainers
```

Reply:
tell me how they look :)

Reply:
Smashing :tada: 

![](https://media.giphy.com/media/Q8fN8ADCh5UWI/giphy.gif)